src/api-guides/e-receipts.markdown

```diff
@@ -160,14 +159,14 @@ To [revoke](/api-reference/authentication/apidoc.html#revoke_token) a user&#39;s
 
 ### <a name="being_geo_aware"></a> Being Geo Aware
 
-#### Concur has multiple datacenters:
+#### SAP Concur Has Multiple Datacenters
 
 US = `https://us.api.concursolutions.com`
 
 EMEA = `https://emea.api.concursolutions.com`
 
 China = `https://cn.api.concurcdc.cn`
 
-You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly.  If there is a case where you will not / do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user.  Sample of Error below:
+You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly. If there is a case where you will not/do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user. Sample of Error below:
```

Comment:
This: `&#39;` was leftover from someone who didn't know the difference between a tick mark and a single quote on their keyboard. Please convert all of these to single quotes when you find them.